cfgov/v1/atomic_elements/organisms.py

```diff
@@ -141,6 +149,32 @@ class Meta:
         template = '_includes/organisms/table.html'
         label = ' '
 
+class AtomicTableInput(TableInput):
+    _id = 'table-block'
+    def render(self, name, value, attrs=None):
+        attrs.update({'id': self._id})
+        # Calling the grandparents render method and bypassing TableInputs,
+        # in order to control how we render the form.
```

Comment:
I notice that inheriting from `TableInput` as you do doesn't actually add much since you're overloading both `render` and `render_js_init`. Does it make sense or simplify to just inherit directly from `WidgetFromScript` and `forms.HiddenInput` as `TableInput` does [here](https://github.com/torchbox/wagtail/blob/v1.5.3/wagtail/contrib/table_block/blocks.py#L14)?